packages/react-client/src/ReactFlightClient.js

```diff
@@ -341,6 +341,11 @@ export type FindSourceMapURLCallback = (
 
 export type DebugChannelCallback = (message: string) => void;
 
+export interface DebugChannel {
+  hasReadable: boolean;
+  callback?: DebugChannelCallback;
```

Comment:
```suggestion
  callback: DebugChannelCallback | null;
```
for hidden-class discipline